config/config.go

```diff
@@ -111,6 +111,7 @@ type ServerConfig struct {
 	GcpPubsubSubscription string `ini:"gcp_pubsub_subscription"`
 	GcpCredentialsFile    string `ini:"gcp_credentials_file"`
 	GcpRegion             string `ini:"gcp_region"`
+	GcpUsePublicIP        bool   `ini:"gcp_use_public_ip"`
```

Comment:
We also need to add the environment variable for this around here: https://github.com/pganalyze/collector/blob/main/config/read.go#L252 (use the `parseConfigBool` method to handle the boolean value there)